openshift/README.md

```diff
@@ -0,0 +1,87 @@
+# OpenShift Cluster API Operator helm chart
+
+OpenShift (OCP) Cluster API Operator helm chart is based on the k8s-sig cluster-api-operator helm [chart](https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/hack/charts/cluster-api-operator).  The OpenShift helm chart uses the [redhat-registry](https://catalog.redhat.com/software/containers/explore) container images to deploy the cluster-api-operator, cluster-api and cluster-api-aws-providers.
+
+## Install
+
+Add CAPI Operator & cert manager helm repository:
+
+```
+$ helm repo add capi-operator https://raw.githubusercontent.com/openshift/cluster-api-operator/refs/heads/main/openshift
```

Comment:
```
$ helm repo add capi-operator https://raw.githubusercontent.com/openshift/cluster-api-operator/refs/heads/main/openshift
Error: looks like "https://raw.githubusercontent.com/openshift/cluster-api-operator/refs/heads/main/openshift" is not a valid chart repository or cannot be reached: failed to fetch https://raw.githubusercontent.com/openshift/cluster-api-operator/refs/heads/main/openshift/index.yaml : 404 Not Found
```

Reply:
the error cause the PR not merged yet. The URL is based on main branch.